api/proto/core/pipeline/cron/cron.proto

```diff
@@ -47,6 +47,41 @@ service CronService {
       put: "/api/pipeline-crons/{cronID}",
     };
   }
+
+  // edge
+  rpc CronInsertOrUpdate (CronInertOrUpdateRequest) returns (CronInertOrUpdateResponse) {
+    option (google.api.http) = {
+      post: "/api/pipeline-crons/actions/insert-or-update",
+    };
+  }
+  rpc CronDisable (CronDisableRequest) returns (CronDisableResponse) {
+    option (google.api.http) = {
+      post: "/api/pipeline-crons/actions/disable",
+    };
+  }
+  rpc EdgeSyncCron (EdgeCronSyncRequest) returns (EdgeCronSyncResponse) {
+    option (google.api.http) = {
+      post: "/api/pipeline-crons/actions/edge-sync",
```

Comment:
I figured it out.
No need this `edge-sync` API.
Edge PIPELINE just receives cron-create(idempotent) request and handle it. **THIS IS SYNC**.